File: business/Business.java
import java.util.Scanner;

public class Business {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int n = sc.nextInt();
		int[] people = new int[n];
		for (int i = 0; i < n; i++)
			people[i] = sc.nextInt();
        	
		double total = 0;
		for (int i = 0; i < n; i++)
			total += people[i];
		
		for (int i = 0; i < n; i++) {
			int percent = (int) Math.round(100 * people[i] / total);
			System.out.println(percent + "%");
		}
	}
}

File: business/business.c
#include <stdio.h>
#include <math.h>

int main(int argc, char **argv) {
	int n;
	scanf("%d", &n);
	int people[n];
	
	double total = 0;
	for (int i = 0; i < n; i++) {
		int in;
		scanf("%d", &in);
		people[i] = in;
		total += in;
	}

	for (int i = 0; i < n; i++) {
		int percent = (int) round(100 * people[i] / total);
		printf("%d%%\n", percent);
	}
}
